### DISCUSSION TITLE
how to investigate precipitation for a BCS class 2 drug

### DISCUSSION THREAD

**Discussion by amitvtaneja** [Open-Systems-Pharmacology/Forum#1682] [Questions & Problems] [answered]
Dear all,

best wishes for 2024!

I am interested in performing a simulation to investigate precipitation for a particular formulation of a BCS class 2 compound for which i have a PBPK model developed for a solution (no formulation effect). For this, as i understand the dissolution input needs to be as PSD. The PSD  is available in um as  deciles (d10-x1 um,d20-x2 um,d30-x3 um...d90). I am using PKsim v 11.1
My question are 
1. as the PSD is available as volume density, is a conversion to numerical density required?
2. Can the method described by Yuri05 in the thread below be applied? Thus creation of 10 monodisperse formulations each with dose fraction corresponding to the respective number  fraction? 
https://github.com/Open-Systems-Pharmacology/Forum/issues/639
 3. In order to run the simulation assuming precipitation, on inputting PSD as aforesaid, under Compounds, can i simply choose Treated precipitated drug as Insoluble from the drop down menu? Or does the simulation need to be exported to MOBI for this step as mentioned in the post below?
 https://github.com/Open-Systems-Pharmacology/Forum/issues/435 
![image](https://github.com/Open-Systems-Pharmacology/Forum/assets/20029513/89a95c1e-2c0e-4fbc-ad00-c230a5542b16)

thanks in advance

Amit Taneja

**Comment by StephanSchaller** [comment, marked as answer]
Dear Amit,

Thanks for your question.

Can you elaborate on the purpose for modeling the precipitation? 
If you "only" want to account for the solubility threshold, precipitation is accounted for in a static (immediate) relationship.

Otherwise, the PSD model can be used, including options for supersaturation for a slower precipitation process.

The Info in the link on Point 3 is only for characterization / IVIVE from in-vitro.

Point 2 on what conversion is intended is unclear to me. Can you explain?

Best,
Stephan
